src/sidebar/store/modules/test/drafts-test.js

```diff
@@ -225,4 +226,44 @@ describe('Drafts Store', () => {
       assert.deepEqual(store.unsavedAnnotations(), []);
     });
   });
+
+  describe('#deleteNewAndEmptyDrafts2', () => {
```

Comment:
```suggestion
  describe('#deleteNewAndEmptyDrafts', () => {
```

I think this was a typo